# Generated by Django 5.2.1 on 2025-06-30 22:26

from django.db import migrations, models


class Migration(migrations.Migration):

    dependencies = [
        ('users', '0003_alter_bonuscard_options_alter_bonusid_options'),
    ]

    operations = [
        migrations.CreateModel(
            name='CardLoginAttempt',
            fields=[
                ('id', models.BigAutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
                ('barcode_id', models.CharField(max_length=20, verbose_name='Штрих-код')),
                ('ip_address', models.GenericIPAddressField(blank=True, null=True, verbose_name='IP адрес')),
                ('success', models.BooleanField(default=False, verbose_name='Успешная попытка')),
                ('error_message', models.TextField(blank=True, verbose_name='Сообщение об ошибке')),
                ('created_at', models.DateTimeField(auto_now_add=True, verbose_name='Время попытки')),
            ],
            options={
                'verbose_name': 'Попытка входа по карте',
                'verbose_name_plural': 'Попытки входа по карте',
                'ordering': ['-created_at'],
            },
        ),
        migrations.AddField(
            model_name='user',
            name='external_id_1c',
            field=models.CharField(blank=True, max_length=50, null=True, verbose_name='ID в системе 1С'),
        ),
        migrations.AddField(
            model_name='user',
            name='registration_source',
            field=models.CharField(choices=[('app', 'Приложение'), ('1c', '1С система'), ('card', 'Физическая карта')], default='app', max_length=20, verbose_name='Источник регистрации'),
        ),
    ]
